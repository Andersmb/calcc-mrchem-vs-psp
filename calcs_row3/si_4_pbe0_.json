{
  "input": {
    "molecule": {
      "cavity_coords": [
        {
          "center": [
            -0.07035905462414478,
            0.0,
            0.0
          ],
          "radius": 1.46
        },
        {
          "center": [
            0.6535009453758551,
            0.0,
            2.7015
          ],
          "radius": 0.79
        },
        {
          "center": [
            0.6535009453758551,
            -2.33956,
            -1.35075
          ],
          "radius": 0.79
        },
        {
          "center": [
            0.6535009453758551,
            2.33956,
            -1.35075
          ],
          "radius": 0.79
        }
      ],
      "cavity_width": 0.2,
      "charge": 0,
      "coords": [
        {
          "atom": "si",
          "xyz": [
            -0.07035905462414478,
            0.0,
            0.0
          ]
        },
        {
          "atom": "h",
          "xyz": [
            0.6535009453758551,
            0.0,
            2.7015
          ]
        },
        {
          "atom": "h",
          "xyz": [
            0.6535009453758551,
            -2.33956,
            -1.35075
          ]
        },
        {
          "atom": "h",
          "xyz": [
            0.6535009453758551,
            2.33956,
            -1.35075
          ]
        }
      ],
      "multiplicity": 2
    },
    "mpi": {
      "bank_size": 1,
      "numerically_exact": false,
      "shared_memory_size": 10000
    },
    "mra": {
      "basis_order": 9,
      "basis_type": "interpolating",
      "boxes": [
        2,
        2,
        2
      ],
      "corner": [
        -1,
        -1,
        -1
      ],
      "max_scale": 20,
      "min_scale": -5
    },
    "printer": {
      "file_name": "si_4_pbe0_.inp",
      "print_level": 0,
      "print_mpi": false,
      "print_prec": 6,
      "print_width": 75
    },
    "rsp_calculations": {},
    "scf_calculation": {
      "fock_operator": {
        "coulomb_operator": {
          "poisson_prec": 1e-06,
          "shared_memory": true
        },
        "exchange_operator": {
          "exchange_prec": -1.0,
          "poisson_prec": 1e-06
        },
        "kinetic_operator": {
          "derivative": "abgv_55"
        },
        "nuclear_operator": {
          "proj_prec": 1e-06,
          "shared_memory": true,
          "smooth_prec": 1e-06
        },
        "xc_operator": {
          "shared_memory": false,
          "xc_functional": {
            "cutoff": 0.0,
            "functionals": [
              {
                "coef": 1.0,
                "name": "pbe0"
              }
            ],
            "spin": true
          }
        }
      },
      "initial_guess": {
        "file_CUBE_a": "cube_vectors/CUBE_a_vector.json",
        "file_CUBE_b": "cube_vectors/CUBE_b_vector.json",
        "file_CUBE_p": "cube_vectors/CUBE_p_vector.json",
        "file_basis": "initial_guess/mrchem.bas",
        "file_chk": "checkpoint/phi_scf",
        "file_gto_a": "initial_guess/mrchem.moa",
        "file_gto_b": "initial_guess/mrchem.mob",
        "file_gto_p": "initial_guess/mrchem.mop",
        "file_phi_a": "initial_guess/phi_a_scf",
        "file_phi_b": "initial_guess/phi_b_scf",
        "file_phi_p": "initial_guess/phi_p_scf",
        "localize": true,
        "method": "DFT (PBE0)",
        "prec": 0.0001,
        "relativity": "None",
        "restricted": false,
        "screen": 12.0,
        "type": "sad_gto",
        "zeta": 0
      },
      "properties": {
        "dipole_moment": {
          "dip-1": {
            "operator": "h_e_dip",
            "precision": 1e-06,
            "r_O": [
              0.0,
              0.0,
              0.0
            ]
          }
        }
      },
      "scf_solver": {
        "checkpoint": false,
        "derivative": "abgv_55",
        "energy_thrs": 1e-06,
        "file_chk": "checkpoint/phi_scf",
        "final_prec": 1e-06,
        "helmholtz_prec": -1.0,
        "kain": 10,
        "light_speed": -1.0,
        "localize": true,
        "max_iter": 200,
        "method": "DFT (PBE0)",
        "orbital_thrs": 4.9999999999999996e-05,
        "proj_prec": 1e-06,
        "relativity": "None",
        "rotation": 0,
        "shared_memory": true,
        "smooth_prec": 1e-06,
        "start_prec": 1e-06
      },
      "write_orbitals": {
        "file_phi_a": "orbitals/phi_a_scf",
        "file_phi_b": "orbitals/phi_b_scf",
        "file_phi_p": "orbitals/phi_p_scf"
      }
    },
    "schema_name": "mrchem_input",
    "schema_version": 1
  },
  "output": {
    "properties": {
      "center_of_mass": [
        4.0670101639234435e-19,
        -4.986846653734096e-18,
        2.22868621828949e-18
      ],
      "charge": 0,
      "dipole_moment": {
        "dip-1": {
          "magnitude": 0.043884959585899545,
          "r_O": [
            0.0,
            0.0,
            0.0
          ],
          "vector": [
            0.04388495928456637,
            -3.95588590694325e-06,
            3.2861763050728854e-06
          ],
          "vector_el": [
            -0.931591112038548,
            -3.955969378839312e-06,
            3.286093436472015e-06
          ],
          "vector_nuc": [
            0.9754760713231144,
            8.347189606183747e-11,
            8.286860087025616e-11
          ]
        }
      },
      "geometry": [
        {
          "symbol": "Si",
          "xyz": [
            -0.07035905462414478,
            0.0,
            0.0
          ]
        },
        {
          "symbol": "H",
          "xyz": [
            0.6535009453758551,
            0.0,
            2.7015
          ]
        },
        {
          "symbol": "H",
          "xyz": [
            0.6535009453758551,
            -2.33956,
            -1.35075
          ]
        },
        {
          "symbol": "H",
          "xyz": [
            0.6535009453758551,
            2.33956,
            -1.35075
          ]
        }
      ],
      "multiplicity": 2,
      "orbital_energies": {
        "energy": [
          -66.2509499221632,
          -5.336938950810035,
          -3.6757441603779775,
          -3.668539764218707,
          -3.6685397096828356,
          -0.5587041632945441,
          -0.3741100147782086,
          -0.374109787564226,
          -0.22817376809673617,
          -66.24728659092982,
          -5.332348538985459,
          -3.6659016790366117,
          -3.6659015886536843,
          -3.6633153197473547,
          -0.534333147631366,
          -0.368172294988317,
          -0.3681721103682165
        ],
        "occupation": [
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0
        ],
        "spin": [
          "a",
          "a",
          "a",
          "a",
          "a",
          "a",
          "a",
          "a",
          "a",
          "b",
          "b",
          "b",
          "b",
          "b",
          "b",
          "b",
          "b"
        ],
        "sum_occupied": -167.9812415113273
      },
      "scf_energy": {
        "E_ee": 148.95119511336273,
        "E_eext": 0.0,
        "E_el": -306.777941531453,
        "E_en": -724.5034741834393,
        "E_kin": 290.62503440790823,
        "E_next": 0.0,
        "E_nn": 15.658344568539409,
        "E_nuc": 15.658344568539409,
        "E_tot": -291.1195969629136,
        "E_x": -5.341622903675309,
        "E_xc": -16.509073965609385,
        "Er_el": 0.0,
        "Er_nuc": 0.0,
        "Er_tot": 0.0
      }
    },
    "provenance": {
      "creator": "MRChem",
      "mpi_processes": 30,
      "nthreads": 6,
      "routine": "mrchem.x",
      "total_cores": 180,
      "version": "1.1.0-alpha"
    },
    "rsp_calculations": null,
    "scf_calculation": {
      "initial_energy": {
        "E_ee": 148.72595302110034,
        "E_eext": 0.0,
        "E_el": -305.203310053149,
        "E_en": -720.6567613664431,
        "E_kin": 288.52144330909573,
        "E_next": 0.0,
        "E_nn": 15.658344568539409,
        "E_nuc": 15.658344568539409,
        "E_tot": -289.5449654846096,
        "E_x": -5.3274495862157245,
        "E_xc": -16.466495430686237,
        "Er_el": 0.0,
        "Er_nuc": 0.0,
        "Er_tot": 0.0
      },
      "scf_solver": {
        "converged": true,
        "cycles": [
          {
            "energy_terms": {
              "E_ee": 148.52530460643538,
              "E_eext": 0.0,
              "E_el": -306.719433749901,
              "E_en": -721.2254648229786,
              "E_kin": 287.75405843308624,
              "E_next": 0.0,
              "E_nn": 15.658344568539409,
              "E_nuc": 15.658344568539409,
              "E_tot": -291.0610891813616,
              "E_x": -5.322895644089193,
              "E_xc": -16.45043632235486,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -291.0610891813616,
            "energy_update": 1.516123696752004,
            "mo_residual": 0.15980139545555602,
            "wall_time": 89.086816976
          },
          {
            "energy_terms": {
              "E_ee": 148.7286118767812,
              "E_eext": 0.0,
              "E_el": -306.767072506653,
              "E_en": -722.9859098374211,
              "E_kin": 289.29828834133747,
              "E_next": 0.0,
              "E_nn": 15.658344568539409,
              "E_nuc": 15.658344568539409,
              "E_tot": -291.1087279381136,
              "E_x": -5.330961388314747,
              "E_xc": -16.477101499035818,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -291.1087279381136,
            "energy_update": 0.047638756752007794,
            "mo_residual": 0.049703777580306266,
            "wall_time": 60.703119902
          },
          {
            "energy_terms": {
              "E_ee": 148.83445372416645,
              "E_eext": 0.0,
              "E_el": -306.7746809069238,
              "E_en": -723.8112400301582,
              "E_kin": 290.0294304801225,
              "E_next": 0.0,
              "E_nn": 15.658344568539409,
              "E_nuc": 15.658344568539409,
              "E_tot": -291.1163363383844,
              "E_x": -5.33552765125494,
              "E_xc": -16.491797429799636,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -291.1163363383844,
            "energy_update": 0.007608400270783022,
            "mo_residual": 0.026672482312854767,
            "wall_time": 52.124988122
          },
          {
            "energy_terms": {
              "E_ee": 148.96615488927824,
              "E_eext": 0.0,
              "E_el": -306.7772269411396,
              "E_en": -724.6075076702522,
              "E_kin": 290.7139606408979,
              "E_next": 0.0,
              "E_nn": 15.658344568539409,
              "E_nuc": 15.658344568539409,
              "E_tot": -291.1188823726002,
              "E_x": -5.3410707603239365,
              "E_xc": -16.508764040739592,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -291.1188823726002,
            "energy_update": 0.0025460342158112326,
            "mo_residual": 0.03323192017309034,
            "wall_time": 52.218707134
          },
          {
            "energy_terms": {
              "E_ee": 148.98662553950058,
              "E_eext": 0.0,
              "E_el": -306.7777534310221,
              "E_en": -724.7403583746132,
              "E_kin": 290.8305399756187,
              "E_next": 0.0,
              "E_nn": 15.658344568539409,
              "E_nuc": 15.658344568539409,
              "E_tot": -291.1194088624827,
              "E_x": -5.3423317267397294,
              "E_xc": -16.512228844788513,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -291.1194088624827,
            "energy_update": 0.0005264898825316777,
            "mo_residual": 0.014661960686569676,
            "wall_time": 50.131153235
          },
          {
            "energy_terms": {
              "E_ee": 148.95876159388476,
              "E_eext": 0.0,
              "E_el": -306.77791255764924,
              "E_en": -724.5492037352177,
              "E_kin": 290.6638476235572,
              "E_next": 0.0,
              "E_nn": 15.658344568539409,
              "E_nuc": 15.658344568539409,
              "E_tot": -291.11956798910984,
              "E_x": -5.341692252760946,
              "E_xc": -16.509625787112608,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -291.11956798910984,
            "energy_update": 0.00015912662712480596,
            "mo_residual": 0.006888919100453844,
            "wall_time": 49.145565683
          },
          {
            "energy_terms": {
              "E_ee": 148.95361481016346,
              "E_eext": 0.0,
              "E_el": -306.77793388862705,
              "E_en": -724.4959967780684,
              "E_kin": 290.6151162581637,
              "E_next": 0.0,
              "E_nn": 15.658344568539409,
              "E_nuc": 15.658344568539409,
              "E_tot": -291.11958932008764,
              "E_x": -5.341606275197544,
              "E_xc": -16.5090619036883,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -291.11958932008764,
            "energy_update": 2.133097780188109e-05,
            "mo_residual": 0.0023753072418370245,
            "wall_time": 51.288732713
          },
          {
            "energy_terms": {
              "E_ee": 148.94910447713468,
              "E_eext": 0.0,
              "E_el": -306.7779391294462,
              "E_en": -724.4842071882073,
              "E_kin": 290.607467228076,
              "E_next": 0.0,
              "E_nn": 15.658344568539409,
              "E_nuc": 15.658344568539409,
              "E_tot": -291.1195945609068,
              "E_x": -5.3415378563553295,
              "E_xc": -16.508765790094298,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -291.1195945609068,
            "energy_update": 5.240819177743106e-06,
            "mo_residual": 0.0020910298982325984,
            "wall_time": 52.102736613
          },
          {
            "energy_terms": {
              "E_ee": 148.94834649640063,
              "E_eext": 0.0,
              "E_el": -306.77794040391785,
              "E_en": -724.4897557760278,
              "E_kin": 290.6138011184051,
              "E_next": 0.0,
              "E_nn": 15.658344568539409,
              "E_nuc": 15.658344568539409,
              "E_tot": -291.11959583537845,
              "E_x": -5.341542539469916,
              "E_xc": -16.508789703225887,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -291.11959583537845,
            "energy_update": 1.2744716286761104e-06,
            "mo_residual": 0.0008078242585398351,
            "wall_time": 54.281619304
          },
          {
            "energy_terms": {
              "E_ee": 148.95059786238895,
              "E_eext": 0.0,
              "E_el": -306.77794131899464,
              "E_en": -724.505846800805,
              "E_kin": 290.628007060653,
              "E_next": 0.0,
              "E_nn": 15.658344568539409,
              "E_nuc": 15.658344568539409,
              "E_tot": -291.11959675045523,
              "E_x": -5.341627030626063,
              "E_xc": -16.50907241060546,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -291.11959675045523,
            "energy_update": 9.150767823484784e-07,
            "mo_residual": 0.0008362381041895413,
            "wall_time": 52.885886136
          },
          {
            "energy_terms": {
              "E_ee": 148.95135684875777,
              "E_eext": 0.0,
              "E_el": -306.7779414702746,
              "E_en": -724.5077793126411,
              "E_kin": 290.6292439495855,
              "E_next": 0.0,
              "E_nn": 15.658344568539409,
              "E_nuc": 15.658344568539409,
              "E_tot": -291.1195969017352,
              "E_x": -5.341638820729781,
              "E_xc": -16.50912413524698,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -291.1195969017352,
            "energy_update": 1.512799485681171e-07,
            "mo_residual": 0.00028042283897243087,
            "wall_time": 53.020225747
          },
          {
            "energy_terms": {
              "E_ee": 148.9514048700708,
              "E_eext": 0.0,
              "E_el": -306.777941518001,
              "E_en": -724.504831577136,
              "E_kin": 290.62621020572294,
              "E_next": 0.0,
              "E_nn": 15.658344568539409,
              "E_nuc": 15.658344568539409,
              "E_tot": -291.1195969494616,
              "E_x": -5.3416288421835345,
              "E_xc": -16.509096174475275,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -291.1195969494616,
            "energy_update": 4.772641659656074e-08,
            "mo_residual": 0.0001642656872587621,
            "wall_time": 52.439784245
          },
          {
            "energy_terms": {
              "E_ee": 148.95119511336273,
              "E_eext": 0.0,
              "E_el": -306.777941531453,
              "E_en": -724.5034741834393,
              "E_kin": 290.62503440790823,
              "E_next": 0.0,
              "E_nn": 15.658344568539409,
              "E_nuc": 15.658344568539409,
              "E_tot": -291.1195969629136,
              "E_x": -5.341622903675309,
              "E_xc": -16.509073965609385,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -291.1195969629136,
            "energy_update": 1.3451995073410217e-08,
            "mo_residual": 5.099381396271888e-05,
            "wall_time": 59.708372676
          }
        ],
        "wall_time": 729.177536307
      },
      "success": true
    },
    "schema_name": "mrchem_output",
    "schema_version": 1,
    "success": true
  }
}
