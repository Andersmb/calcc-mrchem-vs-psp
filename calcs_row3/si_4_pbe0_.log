Starting job 5688413 on c1-[26,39],c2-32,c5-[34-36] at Sat May 21 07:55:31 CEST 2022

Copying si_4_pbe0_.out to /cluster/home/ambr/projects/psp/row3/calcs/doublets
Copying si_4_pbe0_.json to /cluster/home/ambr/projects/psp/row3/calcs/doublets

Task and CPU usage stats:
JobID           JobName  AllocCPUS   NTasks     MinCPU MinCPUTask     AveCPU    Elapsed ExitCode 
------------ ---------- ---------- -------- ---------- ---------- ---------- ---------- -------- 
5688413      si_4_pbe0_        180                                             00:15:53      0:0 
5688413.bat+      batch         36        1   00:00:02          0   00:00:02   00:15:53      0:0 
5688413.ext+     extern        180        6   00:00:00          0   00:00:00   00:15:53      0:0 
5688413.0      mrchem.x        180       30   00:15:21         29   00:33:23   00:15:33      0:0 

Memory usage stats:
JobID            MaxRSS MaxRSSTask     AveRSS MaxPages   MaxPagesTask   AvePages 
------------ ---------- ---------- ---------- -------- -------------- ---------- 
5688413                                                                          
5688413.bat+     19332K          0     19332K        0              0          0 
5688413.ext+          0          0          0        0              0          0 
5688413.0      5053196K         29 2938506035        0              4          0 

Disk usage stats:
JobID         MaxDiskRead MaxDiskReadTask    AveDiskRead MaxDiskWrite MaxDiskWriteTask   AveDiskWrite 
------------ ------------ --------------- -------------- ------------ ---------------- -------------- 
5688413                                                                                               
5688413.bat+       16.91M               0         16.91M        0.03M                0          0.03M 
5688413.ext+        0.00M               0          0.00M            0                0              0 
5688413.0           6.77M               9          6.73M       63.09M               20         30.95M 

Job 5688413 completed at Sat May 21 08:11:24 CEST 2022
