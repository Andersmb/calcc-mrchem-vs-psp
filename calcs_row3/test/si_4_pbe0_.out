                                                                           
                                                                           
***************************************************************************
***                                                                     ***
***                                                                     ***
***          __  __ ____   ____ _                                       ***
***         |  \/  |  _ \ / ___| |__   ___ _ __ ___                     ***
***         | |\/| | |_) | |   | '_ \ / _ \ '_ ` _ \                    ***
***         | |  | |  _ <| |___| | | |  __/ | | | | |                   ***
***         |_|  |_|_| \_\\____|_| |_|\___|_| |_| |_|                   ***
***                                                                     ***
***         VERSION            1.1.0-alpha                              ***
***                                                                     ***
***         Git branch         HEAD                                     ***
***         Git commit hash    e7d4b230bb6df8495b9a                     ***
***         Git commit author  Stig Rune Jensen                         ***
***         Git commit date    Sat Mar 19 14:14:51 2022 +0100           ***
***                                                                     ***
***         Contact: luca.frediani@uit.no                               ***
***                                                                     ***
***         Radovan Bast            Magnar Bjorgve                      ***
***         Roberto Di Remigio      Antoine Durdek                      ***
***         Luca Frediani           Gabriel Gerez                       ***
***         Stig Rune Jensen        Jonas Juselius                      ***
***         Rune Monstad            Peter Wind                          ***
***                                                                     ***
***************************************************************************

---------------------------------------------------------------------------

 MPI processes           :        (1 bank)                              30
 OpenMP threads          :                                               6
 Total cores             :                                             175
                                                                           
---------------------------------------------------------------------------

XCFun DFT library Copyright 2009-2020 Ulf Ekstrom and contributors.
See http://dftlibs.org/xcfun/ for more information.

This is free software; see the source code for copying conditions.
There is ABSOLUTELY NO WARRANTY; not even for MERCHANTABILITY or
FITNESS FOR A PARTICULAR PURPOSE. For details see the documentation.
Scientific users of this library should cite
U. Ekstrom, L. Visscher, R. Bast, A. J. Thorvaldsen and K. Ruud;
J.Chem.Theor.Comp. 2010, DOI: 10.1021/ct100117s

---------------------------------------------------------------------------

 MRCPP version         : 1.4.1
 Git branch            : HEAD
 Git commit hash       : 75d41879b1908a94a452
 Git commit author     : Stig Rune Jensen
 Git commit date       : Thu Jan 6 11:38:53 2022 +0100

 Linear algebra        : EIGEN v3.4.0
 Parallelization       : MPI/OpenMP (6 threads)

---------------------------------------------------------------------------


                                                                           
===========================================================================
                         MultiResolution Analysis
---------------------------------------------------------------------------
 polynomial order      : 9
 polynomial type       : Interpolating
---------------------------------------------------------------------------
 total boxes           : 8
 boxes                 : [          2           2           2 ]
 unit lengths          : [   32.00000    32.00000    32.00000 ]
 scaling factor        : [    1.00000     1.00000     1.00000 ]
 lower bounds          : [  -32.00000   -32.00000   -32.00000 ]
 upper bounds          : [   32.00000    32.00000    32.00000 ]
 total length          : [   64.00000    64.00000    64.00000 ]
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                        Initializing Molecule                        ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
===========================================================================
                                 Molecule
---------------------------------------------------------------------------
 Charge                  :                                               0
 Multiplicity            :                                               2
---------------------------------------------------------------------------
    N    Atom            :               x               y               z
---------------------------------------------------------------------------
    0      Si            :       -0.070359        0.000000        0.000000
    1       H            :        0.653501        0.000000        2.701500
    2       H            :        0.653501       -2.339560       -1.350750
    3       H            :        0.653501        2.339560       -1.350750
---------------------------------------------------------------------------
 Center of mass          :        0.000000       -0.000000        0.000000
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                 Computing Ground State Wavefunction                 ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Compute initial orbitals
 Method                  : Diagonalize SAD Hamiltonian
 Precision               : 1.00000e-04
 Screening               : 1.20000e+01 StdDev
 Restricted              : False
 Functional              : LDA (SVWN5)
 AO basis                : 3-21G
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
                            Molecular Orbitals
---------------------------------------------------------------------------
 Alpha electrons         :                                               9
 Beta electrons          :                                               8
 Total electrons         :                                              17
---------------------------------------------------------------------------
    n  Occ Spin          :                                            Norm
---------------------------------------------------------------------------
    0    1    a          :                              9.999999892226e-01
    1    1    a          :                              9.999999229814e-01
    2    1    a          :                              1.000000079835e+00
    3    1    a          :                              1.000000215408e+00
    4    1    a          :                              9.999999845701e-01
    5    1    a          :                              9.999991111648e-01
    6    1    a          :                              1.000000698206e+00
    7    1    a          :                              1.000001816522e+00
    8    1    a          :                              1.000000041906e+00
    9    1    b          :                              9.999999892226e-01
   10    1    b          :                              9.999999229814e-01
   11    1    b          :                              1.000000079835e+00
   12    1    b          :                              1.000000214062e+00
   13    1    b          :                              9.999999845701e-01
   14    1    b          :                              9.999991112100e-01
   15    1    b          :                              1.000000698116e+00
   16    1    b          :                              1.000001816196e+00
===========================================================================


~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Compute initial energy
 Method                  : DFT (PBE0)
 Relativity              : None
 Precision               : 1.00000e-04
 Localization            : On
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
                        Molecular Energy (initial)
---------------------------------------------------------------------------
 Kinetic energy          :            (au)                288.521443309096
 E-N energy              :            (au)               -720.656761366443
 Coulomb energy          :            (au)                148.725953021100
 Exchange energy         :            (au)                 -5.327449586216
 X-C energy              :            (au)                -16.466495430686
 Ext. field (el)         :            (au)                  0.000000000000
---------------------------------------------------------------------------
 N-N energy              :            (au)                 15.658344568539
 Ext. field (nuc)        :            (au)                  0.000000000000
 Tot. Reac. Energy       :            (au)                  0.000000000000
 El. Reac. Energy        :            (au)                  0.000000000000
 Nuc. Reac. Energy       :            (au)                  0.000000000000
---------------------------------------------------------------------------
 Electronic energy       :            (au)               -305.203310053149
 Nuclear energy          :            (au)                 15.658344568539
---------------------------------------------------------------------------
 Total energy            :            (au)             -2.895449654846e+02
                         :      (kcal/mol)             -1.816922075429e+05
                         :        (kJ/mol)             -7.602001968528e+05
                         :            (eV)             -7.878919545091e+03
===========================================================================


===========================================================================
                        Orbital Energies (initial)
---------------------------------------------------------------------------
    n  Occ Spin          :                                         Epsilon
---------------------------------------------------------------------------
    0    1    a          :            (au)                -65.781313966782
    1    1    a          :            (au)                 -5.290613117609
    2    1    a          :            (au)                 -3.627580681765
    3    1    a          :            (au)                 -3.622165822632
    4    1    a          :            (au)                 -3.621373699910
    5    1    a          :            (au)                 -0.553601032465
    6    1    a          :            (au)                 -0.373303207568
    7    1    a          :            (au)                 -0.360122602387
    8    1    a          :            (au)                 -0.216974365739
    9    1    b          :            (au)                -65.777912277900
   10    1    b          :            (au)                 -5.286031565416
   11    1    b          :            (au)                 -3.619756565660
   12    1    b          :            (au)                 -3.618977465965
   13    1    b          :            (au)                 -3.615029619063
   14    1    b          :            (au)                 -0.527780547464
   15    1    b          :            (au)                 -0.363056526681
   16    1    b          :            (au)                 -0.350446253147
---------------------------------------------------------------------------
 Sum occupied            :            (au)               -166.606039318152
===========================================================================


~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Optimize ground state orbitals
 Method                  : DFT (PBE0)
 Relativity              : None
 Checkpointing           : Off
 Max iterations          : 200
 KAIN solver             : 10
 Localization            : First two iterations
 Diagonalization         : Off
 Start precision         : 1.00000e-06
 Final precision         : 1.00000e-06
 Helmholtz precision     : Dynamic
 Energy threshold        : 1.00000e-06
 Orbital threshold       : 5.00000e-05
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
 Iter           MO residual             Total energy                Update
---------------------------------------------------------------------------
    0          4.123106e+00        -289.544965484610         -2.895450e+02
    1          1.598014e-01        -291.061089181362         -1.516124e+00
    2          4.970378e-02        -291.108727938114         -4.763876e-02
    3          2.667248e-02        -291.116336338384         -7.608400e-03
    4          3.323192e-02        -291.118882372600         -2.546034e-03
    5          1.466196e-02        -291.119408862483         -5.264899e-04
    6          6.888919e-03        -291.119567989110         -1.591266e-04
    7          2.375307e-03        -291.119589320088         -2.133098e-05
    8          2.091030e-03        -291.119594560907         -5.240819e-06
    9          8.078243e-04        -291.119595835378         -1.274472e-06
   10          8.362381e-04        -291.119596750455         -9.150768e-07
   11          2.804228e-04        -291.119596901735         -1.512799e-07
   12          1.642657e-04        -291.119596949462         -4.772642e-08
   13          5.099381e-05        -291.119596962914         -1.345200e-08
---------------------------------------------------------------------------
                      SCF converged in 13 iterations!
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                    Printing Molecular Properties                    ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
===========================================================================
                                 Molecule
---------------------------------------------------------------------------
 Charge                  :                                               0
 Multiplicity            :                                               2
---------------------------------------------------------------------------
    N    Atom            :               x               y               z
---------------------------------------------------------------------------
    0      Si            :       -0.070359        0.000000        0.000000
    1       H            :        0.653501        0.000000        2.701500
    2       H            :        0.653501       -2.339560       -1.350750
    3       H            :        0.653501        2.339560       -1.350750
---------------------------------------------------------------------------
 Center of mass          :        0.000000       -0.000000        0.000000
===========================================================================


===========================================================================
                         Molecular Energy (final)
---------------------------------------------------------------------------
 Kinetic energy          :            (au)                290.625034407908
 E-N energy              :            (au)               -724.503474183439
 Coulomb energy          :            (au)                148.951195113363
 Exchange energy         :            (au)                 -5.341622903675
 X-C energy              :            (au)                -16.509073965609
 Ext. field (el)         :            (au)                  0.000000000000
---------------------------------------------------------------------------
 N-N energy              :            (au)                 15.658344568539
 Ext. field (nuc)        :            (au)                  0.000000000000
 Tot. Reac. Energy       :            (au)                  0.000000000000
 El. Reac. Energy        :            (au)                  0.000000000000
 Nuc. Reac. Energy       :            (au)                  0.000000000000
---------------------------------------------------------------------------
 Electronic energy       :            (au)               -306.777941531453
 Nuclear energy          :            (au)                 15.658344568539
---------------------------------------------------------------------------
 Total energy            :            (au)             -2.911195969629e+02
                         :      (kcal/mol)             -1.826803037057e+05
                         :        (kJ/mol)             -7.643343912007e+05
                         :            (eV)             -7.921767448559e+03
===========================================================================


===========================================================================
                         Orbital Energies (final)
---------------------------------------------------------------------------
    n  Occ Spin          :                                         Epsilon
---------------------------------------------------------------------------
    0    1    a          :            (au)                -66.250949922163
    1    1    a          :            (au)                 -5.336938950810
    2    1    a          :            (au)                 -3.675744160378
    3    1    a          :            (au)                 -3.668539764219
    4    1    a          :            (au)                 -3.668539709683
    5    1    a          :            (au)                 -0.558704163295
    6    1    a          :            (au)                 -0.374110014778
    7    1    a          :            (au)                 -0.374109787564
    8    1    a          :            (au)                 -0.228173768097
    9    1    b          :            (au)                -66.247286590930
   10    1    b          :            (au)                 -5.332348538985
   11    1    b          :            (au)                 -3.665901679037
   12    1    b          :            (au)                 -3.665901588654
   13    1    b          :            (au)                 -3.663315319747
   14    1    b          :            (au)                 -0.534333147631
   15    1    b          :            (au)                 -0.368172294988
   16    1    b          :            (au)                 -0.368172110368
---------------------------------------------------------------------------
 Sum occupied            :            (au)               -167.981241511327
===========================================================================


===========================================================================
                           Dipole Moment (dip-1)
---------------------------------------------------------------------------
 r_O                     :        0.000000        0.000000        0.000000
---------------------------------------------------------------------------
 Electronic vector       :       -0.931591       -0.000004        0.000003
 Magnitude               :            (au)                        0.931591
                         :         (Debye)                        2.367868
---------------------------------------------------------------------------
 Nuclear vector          :        0.975476        0.000000        0.000000
 Magnitude               :            (au)                        0.975476
                         :         (Debye)                        2.479413
---------------------------------------------------------------------------
 Total vector            :        0.043885       -0.000004        0.000003
 Magnitude               :            (au)                        0.043885
                         :         (Debye)                        0.111544
===========================================================================


                                                                           
                                                                           
***************************************************************************
***                                                                     ***
***                            Exiting MRChem                           ***
***                                                                     ***
***                       Wall time :  0h 15m 21s                       ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
